web/src/gear_menu.ts

```diff
@@ -94,6 +94,7 @@ export function version_display_string() {
         const display_version = page_params.zulip_merge_base
             .replace(/\+git.*/, "")
             .replace(/-dev.*/, "-dev");
+
```

Comment:
Can you remove the newlines added here?  (You might need to adjust your editor, which I imagine added them automatically).

Reply:
Okay then i'll revert it.

Reply:
@timabbott done